Source/Turbo Navigator/TurboNavigator.swift

```diff
@@ -24,14 +24,14 @@ public class TurboNavigator {
         }
     }
 
-    /// Default initializer requiring preconfigured `Session` instances.
+    /// Internal initializer requiring preconfigured `Session` instances.
     ///
     /// User `init(pathConfiguration:delegate:)` to only provide a `PathConfiguration`.
     /// - Parameters:
     ///   - session: the main `Session`
     ///   - modalSession: the `Session` used for the modal navigation controller
     ///   - delegate: _optional:_ delegate to handle custom view controllers
-    public init(session: Session, modalSession: Session, delegate: TurboNavigatorDelegate? = nil) {
+    init(session: Session, modalSession: Session, delegate: TurboNavigatorDelegate? = nil) {
```

Comment:
Having this in between two public functions feels weird. Can we move it near line 120 with the internal members?